File: app/src/main/java/com/devative/littledoor/verticalweekcalendar/interfaces/DateWatcher.java
package com.devative.littledoor.verticalweekcalendar.interfaces;


import com.devative.littledoor.verticalweekcalendar.controller.VerticalWeekAdapter;

public interface DateWatcher {
    int getStateForDate(int year, int month, int day, VerticalWeekAdapter.DayViewHolder view);
    void onDateSelection(int year, int month, int day);
}

File: app/src/main/java/com/devative/littledoor/verticalweekcalendar/interfaces/DateWatcher.kt
package com.devative.littledoor.verticalweekcalendar.interfaces

import com.devative.littledoor.verticalweekcalendar.controller.VerticalWeekAdapter.DayViewHolder

interface DateWatcher {
    fun getStateForDate(year: Int, month: Int, day: Int, view: DayViewHolder?): Int
    fun onDateSelection(year: Int, month: Int, day: Int)
}